src/Native/Silk.NET.Assimp.Native/Silk.NET.Assimp.Native.csproj

```diff
@@ -20,12 +20,15 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <IncludeContentInPack>true</IncludeContentInPack>
   </PropertyGroup>
-  
+
   <ItemGroup>
     <Content Include="build\net461\$(PackageId).targets" PackagePath="build\net461\$(PackageId).targets" />
+    <Content Include="runtimes\linux-arm\native\libassimp.so.5" PackagePath="runtimes\linux-arm\native\libassimp.so.5" />
```

Comment:
Future comment: should we just improve this so it does a common glob in common.props? I can totally see us forgetting to add a file here one day.

Reply:
Yes, that's on the list of improvements I plan to make.

Reply:
Awesome.